core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiService.scala

```diff
@@ -44,6 +50,18 @@ trait SubmissionApiService extends UserInfoDirectives {
           }
         }
       } ~
+      path("workspaces" / Segment / Segment / "retrySubmission" / Segment) { (workspaceNamespace, workspaceName, submissionId) => // keeping all logic here and slowly moving out
+        post {
+          val submissionStatus = workspaceServiceConstructor(userInfo).getSubmissionStatus(WorkspaceName(workspaceNamespace, workspaceName), submissionId)
+          submissionStatus.onComplete {
```

Comment:
when/if this code moves into `WorkspaceService`, look at using the `withSubmission` helper method